lib/realtime/tenants/repo/migrations/20231018144023_create_channels.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Realtime.Tenants.Migrations.CreateChannels do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  def change do
+    create table(:channels, prefix: "realtime") do
```

Comment:
Need to check but I think with self-hosted we have the "_realtime" schema too. Might need to make this dynamic.

Reply:
Ah yeah[ our metadata db uses "_realtime"](https://github.com/supabase/realtime/blob/5c7bdf802592c97f3846585673ffcd16c3579618/README.md?plain=1#L67) but we create it with self-hosted anyways.

Reply:
will change 👍

Reply:
No no we use the `realtime` schema on tenants not `_realtime`.